app/models/school.rb

```diff
@@ -0,0 +1,13 @@
+class School < ApplicationRecord
+  belongs_to :local_authority, required: true
```

Comment:
Since Rails 5 `belongs_to` associations are required by default so I think the `required` and the validation below are redundant.